test/log.spec.js

```diff
@@ -173,23 +173,20 @@ apis.forEach((IPFS) => {
       })
 
       it('returns a nicely formatted string', () => {
-        assert.equal(log.toString(), expectedData)
+        assert.strictEqual(log.toString(), expectedData)
       })
     })
 
     describe('get', async () => {
       let log
 
-      const expectedData = { 
+      const expectedData = {
         hash: 'QmXMNDsmtQHW92TSQBnS6zmgZwHXBv9cS65QvziowsfeLo',
         id: 'AAA',
         payload: 'one',
         next: [],
         v: 0,
-        clock: { 
-          id: 'AAA',
-          time: 1,
-        },
+        clock: new Clock('AAA', 1)
```

Comment:
Let' keep the clock's data as it was. The idea here is to make it easy to compare the "pure" data structure and shape and using the Clock here obfuscates that.

Reply:
The problem here is that then the assertion starts to fail because the clock is a simple object and not a LamportClock

![image](https://user-images.githubusercontent.com/1891013/44604538-9af15e80-a7bc-11e8-99ee-0c9541a70468.png)